neutrino.go

```diff
@@ -476,6 +488,23 @@ func (sp *ServerPeer) OnRead(_ *peer.Peer, bytesRead int, msg wire.Message,
 			}
 		}(subscription)
 	}
+	for subscription := range sp.recvSubscribers2 {
+		// Quickly determine if this subscription has been canceled, if
+		// so delete it.
+		select {
+		case <-subscription.quitChan:
+			delete(sp.recvSubscribers2, subscription)
+			continue
+		default:
+		}
+
+		go func(subscription msgSubscription) {
```

Comment:
Don't we want to ensure these messages are delivered in order to the subscriber? I guess the same goes for the above.

Reply:
Doesn't really matter, as it is handled by response handler code